src/ui/common/attributes/AttributeHypeLongMonoTextSettings.js

```diff
@@ -2,12 +2,19 @@ import React from 'react';
 import { Field } from 'redux-form';
 import { FormattedMessage } from 'react-intl';
 import { Row, Col } from 'patternfly-react';
+import { isUndefined } from 'lodash';
+import { isNumber } from '@entando/utils';
 import RenderTextInput from 'ui/common/form/RenderTextInput';
 import FormLabel from 'ui/common/form/FormLabel';
-import { isNumber } from '@entando/utils';
+
+const validateIsNumber = (value) => {
```

Comment:
This is not needed anymore after merging https://github.com/entando/frontend-libraries/pull/6